src/main/java/org/corrige/ai/validations/exceptions/TopicNotExistsException.java

```diff
@@ -0,0 +1,14 @@
+package org.corrige.ai.validations.exceptions;
+
+public class TopicNotExistsException extends Exception{
```

Comment:
add the exception to exception translator class with the our pattern of construct error or use @ResponseStatus annotation, on the top of class
like this:
`@ResponseStatus(HttpStatus.NOT_FOUND)`